core/chains/evm/gas/fixed_price_estimator.go

```diff
@@ -89,6 +107,28 @@ func (f *fixedPriceEstimator) GetDynamicFee(_ context.Context, originalGasLimit
 	}, chainSpecificGasLimit, nil
 }
 
-func (f *fixedPriceEstimator) BumpDynamicFee(_ context.Context, originalFee DynamicFee, originalGasLimit uint32, maxGasPriceWei *assets.Wei, _ []EvmPriorAttempt) (bumped DynamicFee, chainSpecificGasLimit uint32, err error) {
-	return BumpDynamicFeeOnly(f.config, f.bhConfig.EIP1559FeeCapBufferBlocks(), f.lggr, f.config.TipCapDefault(), nil, originalFee, originalGasLimit, maxGasPriceWei)
+func (f *fixedPriceEstimator) BumpDynamicFee(
+	_ context.Context,
+	originalFee DynamicFee,
+	originalGasLimit uint32,
+	maxGasPriceWei *assets.Wei,
+	_ []EvmPriorAttempt,
+) (bumped DynamicFee, chainSpecificGasLimit uint32, err error) {
+
+	return BumpDynamicFeeOnly(
+		f.config,
+		f.bhConfig.EIP1559FeeCapBufferBlocks(),
+		f.lggr,
+		f.config.TipCapDefault(),
+		nil,
+		originalFee,
+		originalGasLimit,
+		maxGasPriceWei,
+	)
 }
+
+func (f *fixedPriceEstimator) Name() string                                          { return f.lggr.Name() }
+func (f *fixedPriceEstimator) Ready() error                                          { return nil }
+func (f *fixedPriceEstimator) HealthReport() map[string]error                        { return map[string]error{} }
+func (f *fixedPriceEstimator) Close() error                                          { return nil }
+func (f *fixedPriceEstimator) OnNewLongestChain(_ context.Context, _ *evmtypes.Head) {}
```

Comment:
nit: just wondering, why all the reorganization of the code?
it makes it seem like your PR is bigger than it actually is 🤷🏼

Reply:
I felt that the fee estimator was structured in a pretty confusing way. Felt that way when i was generalising the fee estimator initially, but now as well 🙃 

This style where we have the non essential methods at the bottom is used by the headtracker as well https://github.com/smartcontractkit/chainlink/blob/e42536d934e8c25a1b95d1baf2a2702cd7429595/core/chains/evm/headtracker/head_tracker.go#L50

Reply:
also expanded the other functions for readability due to long input parameters which can be confusing with named return values